seed/building_sync/building_sync.py

```diff
@@ -90,6 +90,11 @@ class BuildingSync(object):
                 "required": True,
                 "type": "string",
             },
+            "ubid": {
+                "path": "auc:Sites.auc:Site.auc:Buildings.auc:Building.auc:Ubid",
```

Comment:
According to the schema, the UBID is under Identifier Label (//element(*,auc:SiteType)/auc:Buildings/auc:Building/auc:PremisesIdentifiers/auc:PremisesIdentifier/auc:IdentifierLabel):

![image](https://user-images.githubusercontent.com/1907354/68133760-39f40900-fede-11e9-9ef7-00ba80764aec.png)

Which would make this looks like these other mapping entries:

![image](https://user-images.githubusercontent.com/1907354/68133889-77589680-fede-11e9-8579-a7acd0f3a568.png)

Reply:
Don't believe i did. There isn't any if i remember right but I modified one locally for testing.

Reply:
ok thanks, I'll correct that.